Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs

```diff
@@ -222,8 +222,8 @@ public override object SetReportBusinessFlow(Context context, bool offlineMode,
             int ChildExecutedItemsCountActivity = 0;
             int ChildPassedItemsCountActivity = 0;
 
-            ChildExecutableItemsCountActivity = context.BusinessFlow.Activities.Count;
-            ChildExecutedItemsCountActivity = ChildExecutableItemsCountActivity - context.BusinessFlow.Activities.Where(ac => ac.Status == eRunStatus.Pending || ac.Status == eRunStatus.Skipped || ac.Status == eRunStatus.Blocked).Count();
+            ChildExecutableItemsCountActivity = context.BusinessFlow.Activities.Count(x => x.Active == true);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4629074)